app/managers.py

```diff
@@ -0,0 +1,41 @@
+import sqlite3
+
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("cinema")

```

Comment:
You forgot to add a file extension, for example `cinema.db3`

Reply:
it doesnt work if i change the extension for file

Reply:
Okey)

Reply:
Why doesn't it work? It should. Probably you configured the extension manually. You can try to delete your db and create a new one. Because adding an extension is a good thing